src/components/Sections.js

```diff
@@ -199,75 +224,90 @@ export const DebugLogSettings = () => {
                   "trustedlogin-vendor"
                 )}
                 <code>wp-content/uploads/trustedlogin-logs</code>
-                {/* TODO: Make this path live-updating */}
+                {phpConstantIsSet ? (
```

Comment:
@zackkatz There is a message set if TRUSTEDLOGIN_DEBUG has a value. Also the toggle doesn't work.